enclave_manager/setup.py

```diff
@@ -78,10 +78,11 @@
     os.path.join(enclave_bridge_wrapper_path, 'enclave_info.cpp'),
     os.path.join(enclave_bridge_wrapper_path, 'signup_info.cpp'),
     os.path.join(enclave_bridge_wrapper_path, 'signup_info_singleton.cpp'),
-    # TODO: Move signup_info_kme.cpp to KME enclave manager and 
-    # move move signup_info_wpe.cpp to WPE enclave manager
+    # TODO: Move *_kme.cpp to KME enclave manager and
+    # move *_wpe.cpp to WPE enclave manager
     os.path.join(enclave_bridge_wrapper_path, 'signup_info_kme.cpp'),
     os.path.join(enclave_bridge_wrapper_path, 'signup_info_wpe.cpp')
```

Comment:
Syntax error here, missing comma, causing the build to fail.

Reply:
Fixed.